app/sim_test.go

```diff
@@ -16,11 +16,10 @@ import (
 	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 
-	magpieTypes "github.com/desmos-labs/desmos/x/magpie/types"
-	postsTypes "github.com/desmos-labs/desmos/x/posts/types"
 	profilesTypes "github.com/desmos-labs/desmos/x/profiles/types"
-	relationshipsTypes "github.com/desmos-labs/desmos/x/relationships/types"
-	reportsTypes "github.com/desmos-labs/desmos/x/reports/types"
+	postsTypes "github.com/desmos-labs/desmos/x/staging/posts/types"
+	relationshipsTypes "github.com/desmos-labs/desmos/x/staging/relationships/types"
+	reportsTypes "github.com/desmos-labs/desmos/x/staging/reports/types"
```

Comment:
Can we use the same naming conventions as other imports (all lowercase)? `profilesTypes` -> `profilestypes`